capsules/core/src/driver.rs

```diff
@@ -67,6 +67,7 @@ pub enum NUM {
     SoundPressure         = 0x60006,
     AirQuality            = 0x60007,
     Pressure              = 0x60008,
+    Nmea                  = 0x60009,
```

Comment:
I'm sorry to be pedantic here, but NMEA is not a sensor. It is a hardware bus, quite similar to CAN etc in purpose, so it's driver number should be with the HW bus prefix (e.g. `0x2007`).

GPS/Location, wind speed, etc... those are sensors. They may operate _on top_ of NMEA (or may not), but would have a sensor-specific interface (`get_location`, rather than `read_sentence`).